src/lib/model/index.js

```diff
@@ -23,6 +23,7 @@ import { createAddressConfig } from './address';
 import { createInterface } from './interfaces';
 import { createConnection, mergeConnection } from './connections';
 import { createRoute } from './routes';
+import { createDnsSettings } from './dnsSettings';
```

Comment:
np: what about using `lib/model/dns`? You could put more DNS-related stuff there. Just a suggestion.